server/controllers/bookings/getAllBookings.js

```diff
@@ -2,22 +2,23 @@ import jwt from 'jsonwebtoken';
 import resPonse from '../../helpers/responses/response';
 import Book from '../../models/bookings';
 
-const getAllBookings = (req, res) => {
-  try {
-    const getUser = jwt.decode(req.headers.authorization);
-    if (getUser.isAdmin === true) {
-      const allBookings = Book.bookingDatabase;
-      if (allBookings.length === 0) {
-        return resPonse.errorMessage(res, 404, 'No bookings made at the moment');
-      } resPonse.successData(res, 200, 'All trips successfully fetched', allBookings);
-    }
-    const forUser = Book.getUserBookings(getUser.id);
-    if (forUser.length === 0) {
-      return resPonse.errorMessage(res, 200, 'You have made no bookings yet');
-    } resPonse.successData(res, 200, forUser);
-  } catch (error) {
-    resPonse.errorMessage(res, 500, error.message);
+const book = new Book();
+const getAllBookings = async (req, res) => {
+  // try {
+  const getUser = jwt.decode(req.headers.authorization);
+  if (getUser.isAdmin === true) {
+    const allBookings = await book.getAtllbookings;
+    if (allBookings.rowCount === 0) {
+      return resPonse.errorMessage(res, 404, 'No bookings made at the moment');
+    } resPonse.successData(res, 200, 'All bookings successfully fetched', allBookings.rows);
   }
+  const forUser = await book.getUserBookings(getUser.id);
+  if (forUser.length === 0) {
+    return resPonse.errorMessage(res, 200, 'You have made no bookings yet');
+  } resPonse.successData(res, 200, forUser);
+  // } catch (error) {
+  //   resPonse.errorMessage(res, 500, error.message);
```

Comment:
Why are you commenting this out, remove it entirely if you think it doesn't add any value.